extensions/amp-inline-gallery/1.0/README.md

```diff
@@ -216,7 +216,3 @@ Number: ratio of width to height that slides should be displayed in.
 Default: `false`
 
 Boolean attribute indicating whether thumbnails should loop.
-
```

Comment:
This PR and #36021 both have some extraneous edits to this file. Do you think it's worth consolidating changes to this file and the amp-stream-gallery documentation into a single PR?

Reply:
sure

Reply:
closing this PR and merging the commits from this branch into the date-display PR #36021